doc-content/enterprise-only/installation/kie-server-thread-efficiency.adoc

```diff
@@ -0,0 +1,20 @@
+[id='kie-server-thread-efficiency.adoc_{context}']
+= Thread Efficiency
+
+To ensure the optimal number of threads used, set the threading system properties to the number of CPUs in the system + 1.
```

Comment:
To ensure that the optimal number of threads are used, set the value of the threading system property to the sum of the number of CPUs plus one. 
Is the property called 'threading'. If so, code. If only one, then property.